source/overloading.tex

```diff
@@ -3453,8 +3434,20 @@
 Operator functions are inherited in the same manner as other base class
 functions.
 
+\pnum
+An operator function shall declare
+a prefix unary operator,
+a binary operator,
+a simple assignment,
+a function call,
+a subscripting,
+a class member access using \tcode{->},
+an increment or decrement,
+or a literal operator.
```

Comment:
Looks like, per [over.oper]/1, literal operators are not operator functions. Maybe we should move the description of them out of [over.oper] too?

Reply:
That sounds like a good idea. In fact, [over.literal] has nothing to do with [over.oper].

Reply:
Fixed.